README.md

```diff
@@ -14,11 +14,113 @@ composer require alleyinteractive/wp-widget-control
 
 ## Usage
 
-Use this package like so:
+WP Widget Control lets you programmatically manage WordPress sidebars and
+widgets. This can be useful for setting up default widget configurations,
+managing widget curation with code for end to end testing, or simply
+maintaining widget state in a more structured way.
+
+When you have to manage widgets with code, WordPress doesn't provide a great way
+to do that. WP Widget Control fills this gap by providing a simple API for
+managing widgets programmatically.
+
+Here are some common usage patterns:
+
+### Retrieve a Sidebar
+
+```php
+use Alley\WP\Widget_Control\Storage\Sidebar;
```

Comment:
Related to one of my questions, is there a scenario where a sidebar or widget is not stored?